extensions/ql-vscode/src/config.ts

```diff
@@ -718,3 +718,12 @@ const LLM_GENERATION = new Setting("llmGeneration", DATA_EXTENSIONS);
 export function showLlmGeneration(): boolean {
   return !!LLM_GENERATION.getValue<boolean>();
 }
+
+/**
+ * A flag indicating whether to make code search for feeding variant analysis lists accessible.
+ */
+const CODE_SEARCH = new Setting("codeSearch", ROOT_SETTING);
```

Comment:
Would it make sense to make this a variant analysis setting rather than a root level one?

Reply:
I don't know. On https://github.com/github/vscode-codeql/pull/2476#issuecomment-1576438499 we thought it best to consider it a piece of "databases" functionality rather than variant analysis. So my initial thought was that this setting should also mirror that. Happy to be persuaded otherwise though.

Reply:
Removed this part of code because it's not needed